src/renderer/components/atoms/text/types.ts

```diff
@@ -0,0 +1,53 @@
+import type { ReactNode } from 'react';
+
+export type TextTag = 'h1' | 'h2' | 'h3' | 'h4' | 'span' | 'p';
+export type TextUsage =
+  | 'big-title'
+  | 'typical-title'
+  | 'section-title'
+  | 'sub-title'
+  | 'content-body'
+  | 'body-small'
+  | 'title'
+  | 'chapter'
+  | 'chapter-section'
+  | 'content-title'
+  | 'content'
+  | 'content-small'
+  | 'error'
+  | 'quote';
+export type TextVariant = 'bold' | 'normal' | 'italic' | '';
+export type TextColor =
+  | 'primary'
+  | 'secondary'
+  | 'blueWhite'
+  | 'redWhite'
+  | 'warning'
+  | 'error'
+  | 'success'
+  | '';
```

Comment:
I'm assuming these props are optional, right? That's why you added a `''` as one of the possible values?

Reply:
Yes, but I changed it because when  props of this type are undefined, there are default values